contracts/staking/stakeManager/StakeManager.sol

```diff
@@ -346,20 +297,6 @@ contract StakeManager is IStakeManager, StakeManagerStorage {
         logger.logClaimRewards(validatorId, amount, totalRewardsLiquidated);
     }
 
-    // returns valid validator for current epoch
-    function getCurrentValidatorSet() public view returns (uint256[] memory) {
```

Comment:
Are there any other methods to get all validators?

Reply:
This wasn't used by any contract and the external party can read the blockchain and using same as this function.